inst/pandoc_templates/jolla-blue.html

```diff
@@ -12,6 +12,21 @@
     <style type="text/css">body {background-color: #3A66B7;font-family: 'Merriweather', serif;color: #FEFEFA;}</style>
 
     <title>$title$</title>
+    
+    $if(meta)$
+    <meta name="description" content="$meta.description$"></meta>
+      <meta name="twitter:description" content="$meta.description$"></meta>
+      <meta name="og:description" content="$meta.description$"></meta>
+      <meta name="twitter:title" content="$title$"></meta>
+      <meta name="og:title" content="$title$"></meta>
+      <meta name="twitter:url" content="$meta.url$"></meta>
+      <meta property="og:url" content="$meta.url$"></meta>
+      <meta name="twitter:image:src" content="$meta.url$/$image$"></meta>
+      <meta property="og:image" content="$meta.url$/$image$"></meta>
```

Comment:
In addition to `og:image`, I recommend that postcards also support the `og:image:alt` and `twitter:image:alt` tags as well. One option would be to add `image_url` or `meta.image_url` (or both) to the YAML:

```yaml
meta:
  description: "floatplane data artist"
  url: "example.com"
  twitter: "yourname"
  image: https://example.com/social.png
  image_alt: Short description of social card image.
image: avatar.png
image_alt: Short description of avatar image.
```

The two tags to add would be something like

```html
<meta name="twitter:image:alt" content="$if(meta.image_alt)$$meta.image_alt$$else$$image_alt$$endif$"/>
<meta property="og:image:alt" content="$if(meta.image_alt)$$meta.image_alt$$else$$image_alt$$endif$"/>
```

Reply:
Relatedly, `image_alt` could be added here https://github.com/seankross/postcards/blob/daad882a00e548f99217903da58ed69c4bca7e00/inst/pandoc_templates/solana.html#L68